autogen/agentchat/contrib/vectordb/pgvectordb.py

```diff
@@ -32,10 +32,11 @@ class Collection:
         client: The PGVector client.
         collection_name (str): The name of the collection. Default is "documents".
         embedding_function (Callable): The embedding function used to generate the vector representation.
+            Default is None. SentenceTransformer("all-MiniLM-L6-v2") will be used when None.
```

Comment:
SentenceTransformer("all-MiniLM-L6-v2").encode will  be used

Reply:
I have made this correction.